src/Input/InputController.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2012-2015 Falltergeist Developers.
+ *
+ * This file is part of Falltergeist.
+ *
+ * Falltergeist is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Falltergeist is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Falltergeist.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef FALLTERGEIST_INPUT_INPUTCONTROLLER_H
+#define FALLTERGEIST_INPUT_INPUTCONTROLLER_H
+
+// C++ standard includes
+
+// Falltergeist includes
+
+// Third party includes
+
+namespace Falltergeist
+{
+
+namespace Lua
+{
+    class Script;
+}
+
+class Mouse;
+
+class InputController
```

Comment:
I think that we shouldn't allow direct access to some engine objects(like mouse) from Lua scripts.
Also we need to do some checks for parameters from scripts.
And this class could do this. We can use more suitable name if you have one.